build/README.md

```diff
@@ -0,0 +1,40 @@
+# About 
+
+Docker Script for Building IOT-platform-SDK repository. Using this script you can build the local copy of the repository as well as the latest upstream of the repository.
+
+## Prerequisites
+
+- Operating system: **Ubuntu 18.04.**
+
+- Docker engine : **18.06.0**
+
+- Docker-compose : **1.23.2**
+
+
+
+## Usage
+
+When you want to build a local copy of the repository.
+
+``` sudo docker-compose up --build ```
+
+When you want to build the latest upstream of the repository.
+
+``` sudo use_remote=1 docker-compose up --build ```
+
+You also have the option to change the remote repository address as well as the remote repository branch in build.sh file.
+
+    REMOTE_URL=link-to-your-fork
+    REMOTE_BRANCH=branch-name
+
+## Expected Outcome
+As the docker script finishes its execution, the ```.war``` files of the IOT-platform-SDK will be present in ```<iot-platform-sdk>/demo/{ocs,ops,to0scheduler}``` folders.
+
+## Updating Proxy Info (Optional )
+If you are working behind a proxy network, ensure that both http and https proxy variables are set.
+
+    export http_proxy=http-proxy-host:http-proxy-port
+    export https_proxy=https-proxy-host:https-proxy-port 
+
+## Contributing
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
```

Comment:
"Contributing" section is not required here, can be removed or added to top-level README file.

In addition, please check whether https://secure-device-onboard.github.io/docs/community/ provides sufficient information about process for contribution.

Reply:
Will remove the Contribution section ;